crates/transaction-emitter/src/lib.rs

```diff
@@ -41,6 +44,7 @@ const MAX_TXN_BATCH_SIZE: usize = 100;
 const MAX_TXNS: u64 = 1_000_000;
 const SEND_AMOUNT: u64 = 1;
 const TXN_EXPIRATION_SECONDS: u64 = 180;
```

Comment:
We might do not need this const since it is only used blow

Reply:
I still think its good to define consts for magic numbers, gives them a bit more meaning IMO